pkg/proto/agentbaker/v1/config.proto

```diff
@@ -8,56 +8,58 @@ message Configuration {
   
   string cluster_certificate_authority = 1;
 
-  // only required until Secure TLS bootstrapping in place
-  // would use kubelet identity after that.
```

Comment:
client-go credential plugin, not kubelet identity :)

Reply:
I think we'll also need the server ID for this we were speaking about with Alex/Bryce - though it'll end up being an override I think